content/en/ref/python/sdk/data-types/Image.md

```diff
@@ -9,10 +9,10 @@ data_type_classification: class
 
 
 
-## <kbd>class</kbd> `Image`
+<kbd>class</kbd> `Image`
```

Comment:
Formatting is correct. Undo changes. (Note: Anything in content/en/ref/python/* autogenerated. To make changes, make a PR in wandb/wandb).